Makefile

```
@@ -1093,7 +1093,7 @@ $(TARBALL): release-only doc-only
 	find $(TARNAME)/deps/v8/test -type f ! -regex '.*/test/torque/.*' | xargs $(RM)
 	find $(TARNAME)/deps/zlib/contrib/* -type d ! -regex '.*/contrib/optimizations$$' | xargs $(RM) -r
 	find $(TARNAME)/ -name ".eslint*" -maxdepth 2 | xargs $(RM)
-	find $(TARNAME)/ -type l | xargs $(RM) # annoying on windows
+	find $(TARNAME)/ -type l | xargs $(RM) # annoying on Windows
```

I'm not sure this comment has much value, maybe we should consider removing it instead.
```suggestion
	find $(TARNAME)/ -type l | xargs $(RM)
```

---

@aduh95 Good point. Comment removed.